.github/actions/run-e2e-tests/action.yaml

```diff
@@ -0,0 +1,56 @@
+name: "Run SDK E2E tests"
+description: |
+  Run the SDK E2E tests against a local testnet from the latest Aptos CLI
+
+# Currently no indexer tests or tests against local testnets from production branches,
+# we just use whatever is in the latest CLI.
+
+runs:
+  using: composite
+  steps:
+    # Install node and pnpm.
+    - uses: actions/setup-node@v3
+      with:
+        node-version-file: .node-version
+        registry-url: "https://registry.npmjs.org"
+    - uses: pnpm/action-setup@v2
+      with:
+        version: 8.9.0
+
+    # Run package install. If install fails, it probably means the updated lockfile was
+    # not included in the commit.
+    - run: pnpm install --frozen-lockfile
+      shell: bash
+
+    # Install the CLI.
+    - run: pnpm install -g @aptos-labs/aptos-cli
+      shell: bash
+
+    # Run a local testnet in the background.
+    - run: aptos node run-local-testnet --force-restart --assume-yes --with-faucet &
```

Comment:
interesting... I thought we would need to `npx aptos....`

Reply:
In this case since runners are transient I just install the CLI globally.